pixy/httpapi.go

```diff
@@ -0,0 +1,131 @@
+package pixy
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/gorilla/mux"
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/mailgun/manners"
+)
+
+const (
+	NetworkTCP  = "tcp"
+	NetworkUnix = "unix"
+
+	hContentLength = "Content-Length"
+
+	fTopicName  = "topic"
+	fRoutingKey = "key"
```

Comment:
I think this use of [Hungarian Notation](http://en.wikipedia.org/wiki/Hungarian_notation) is appropriate and improves developer understanding of the use of these constants.